modular_skyrat/code/game/turfs/open/floor/planetfloor.dm

```diff
@@ -0,0 +1,14 @@
+/turf/open/floor/plating/cobblestone
+	name = "cobblestone"
+	desc = "Stone cobbles set into the ground."
+	icon = 'modular_skyrat/icons/turf/floors/planetfloor.dmi'
+	icon_state = "cobblestone"
+	baseturfs = /turf/open/floor/plating/asteroid
+
+/turf/open/floor/plating/cobblestone/snow
+	name = "snowy cobblestone"
+	desc = "Snnnooooowwwww! Snowy cobblestone."
+	icon_state = "cobblestone_snow"
+
+/turf/open/floor/cobblestone/Initialize()
+	dir = pick(NORTH, SOUTH, EAST, WEST)
```

Comment:
dont list out the cardinals. use GLOB.cardinals instead.

Reply:
It didn't work regardless, not sure what the difference between cardinals and global cardinals is.
Now it's just 1, 2, 4 & 8.

Reply:
It's now picking different icon_states instead of using dir.